File: CSharp/7kyu/IsSquare.cs
using System;

namespace CSharp._7Kyu
{
    public static partial class Kata7
    {
        public static bool IsSquare(int n)
        {
            return Math.Ceiling(Math.Sqrt(n)) == Math.Floor(Math.Sqrt(n));
        }

        public static bool IsSquare2(int n)
        {
            return Math.Sqrt(n) % 1 == 0;
        }
    }
}

File: Python/7kyu/IsSquare.py
import math


def is_square(n):
    return n > -1 and math.sqrt(n) % 1 == 0


def is_squre2(n):
    return n > -1 and n**0.5 % 1 == 0


result = is_square(-1)
print(result)
